browser/static/ably-commonjs.js

```diff
@@ -11442,6 +11442,11 @@ var JSONPTransport = (function() {
 	return JSONPTransport;
 })();
 
+var Types = {
+  ClientOptions: function(obj) { return (obj); }
```

Comment:
Can you please not commit static compiled files until this is ready for release?  Looking through this PR, only three files have actually changed, yet this PR has ten changed files, which I have to look through to make sure I'm not missing anything.

Reply:
I committed those just to test the results of the concatenation. They won't be there when this is ready for review.